src/PKSim.Core/Services/ParameterSetUpdater.cs

```diff
@@ -143,11 +150,16 @@ public ICommand UpdateValuesByName(IEnumerable<IParameter> sourceParameters, IEn
          return updateCommands;
       }
 
-      public ICommand UpdateValue(IParameter sourceParameter, IParameter targetParameter)
+      public ICommand UpdateValue(IParameter sourceParameter, IParameter targetParameter, bool updateParameterOriginId=true)
       {
-         _parameterIdUpdater.UpdateParameterId(sourceParameter, targetParameter);
+         if(updateParameterOriginId)
+            _parameterIdUpdater.UpdateParameterId(sourceParameter, targetParameter);
+
+         var command = _parameterUpdater.UpdateValue(sourceParameter, targetParameter);
+
+         //this line ensure that value origin is updated for all parameters, even the one for which the value were not changed
          targetParameter.ValueOrigin.UpdateFrom(sourceParameter.ValueOrigin);
```

Comment:
With this line, even PARAMETER whose value have not changed will get the updated value origin. I think this is the best way to implement the feature. 
Drawback. In case of update/commit and undo, some ValueOrigin that may have changed without value changed might be outdated. I don't think this happens really often. On the other hand, when a ValueOrigin is updated in say a simulation, it is important to have a way to update it in the building block without having to change the value of the parameter.